cdap-cli/src/main/java/co/cask/cdap/cli/command/HelpCommand.java

```diff
@@ -33,6 +34,8 @@
  */
 public class HelpCommand implements Command {
 
+  private static final int COL_WIDTH = 80;
```

Comment:
I think this should be determined from the current width of the terminal. Not sure if it's possible with jline2 though.